### PR TITLE
Add GPU support

### PR DESCRIPTION
**Pull request by qaisjp**

### CODE REVIEW

File: website/map/static/js/mapp.js [hunk 1]
```diff
@@ -191,7 +191,11 @@ function readyFunction(){
                     const icon = $("<p class=blip><i class=fa></i></p>")
                     const text = $("<p></p>");
 
-                    text.text(cell.hostname);
+                    let gpuSuffix = "";
+                    if (cell.gpu == "True") {
+                        gpuSuffix = " (gpu)"
+                    }
+                    text.text(cell.hostname+gpuSuffix);
```
Comment (neanias) [inline website/map/static/js/mapp.js]:
```suggestion
                    text.text(cell.hostname + gpuSuffix);
```

Comment (neanias) [reply website/map/static/js/mapp.js]: Just for the sake of style.

File: website/map/views.py [hunk 2]
```diff
@@ -437,14 +437,15 @@ def update_schema():
                         raise APIError("[Sheet %s] Invalid value '%s' in rows[%s], expected empty row" % (sheet['name'], cell_value, rownumber))
                     continue
 
-                hostname = cell_value
+                hostname = cell_value.lower()
 
                 if hostname != "" and not dropOnly:
```
Comment (neanias) [inline website/map/views.py:442]: Something to think about: `dropOnly` is not at all Pythonic

Comment (qaisjp) [reply website/map/views.py:442]: should be `drop_only`?

**Comment by neanias** [comment]
Yes. Python identifiers should be in snake case (aptly).

File: website/map/views.py (hunk 2 above)
Comment (qaisjp) [reply website/map/views.py:442]: OK, I wlil fix this the next time I touch the dropOnly code

**Review by neanias — APPROVED** [review]

**Pull request merged**